src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3406,6 +3406,12 @@ void SpellMgr::LoadDbcDataCorrections()
         spellInfo->EffectImplicitTargetB[0] = 0;
     });
 
+    // Flash Freeze Radius - Avoid FPS Drops
+    ApplySpellFix({ 62148 }, [](SpellEntry* spellInfo)
+    {
+        spellInfo->RangeIndex = 100;
```

Comment:
There is no ID 100 in SpellRange.dbc

Reply:
https://gist.github.com/Kitzunu/403ea5c8c0116924c2eb8eefe57420f9 there's all ranges